### PR TITLE
Add Order each Card set by Released Date

### PR DESCRIPTION
**Pull request by Mikechoi78**
I have done two functions.
. Order cards by release_time
. Add "Release Time" line in the back of card

### CODE REVIEW

**Comment by ryanprice** [comment]
Looking good so far. Can you shorten the label to "Released:"    lets see if can we the label and date on 1 line.

also I got this error in console

image

**Comment by Mikechoi78** [comment]
I changed "Release Time:" to "Released:".
I fixed the infinite warning.

File: src/components/CardSets.vue [hunk 1]
```diff
@@ -240,6 +242,17 @@ export default {
           return null;
       }
     },
+    getSortedCards(cards) {
```
Comment (mchigit) [inline src/components/CardSets.vue:245]: We are actually sorting cards by rarity here: https://github.com/ryanprice/Cryptoz.cards-Dapp/blob/zoombies-master/src/store/cardsetStore.js#L87-L95. If we are not sorting by rarity anymore, I think you can just modify this sort comparator to achieve order by release time. Currently the cards in each set are sorted by rarity first and then release time.

**Review by mchigit — COMMENTED** [review]
evyerthing else LGTM

**Comment by ryanprice** [comment]
closing this unmerged

**Pull request closed without merging**